inc/navigation-widget.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Custom Navigation menu widget
+ *
+ * @package uds-wordpress-theme
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+require_once get_template_directory() . '/inc/class-custom-walker-nav-menu.php';
+/**
+ * Customize the navigation menu widget's wrapper and title's wrapper.
+ *
+ * @param args $args Array of nav menu arguments.
+ */
+function uds_wp_theme_custom_widget_title_tag( $args ) {
+	if ( 'Navigation Menu' === $args[0]['widget_name'] ) {
+		$args[0]['before_widget'] = '<nav id="' . $args[0]['widget_id'] . '" class="sidebar accordion" aria-label="Secondary">';
+		$args[0]['after_widget'] = '</nav>';
+		$args[0]['before_title'] = '<div class="nav-text">';
+		$args[0]['after_title']  = '</div>';
+	}
+		return $args;
+}
+add_filter( 'dynamic_sidebar_params', 'uds_wp_theme_custom_widget_title_tag' );
+
+
+/**
```

Comment:
Always wrap these functions with

```
if ( ! function_exists( 'uds_function_name' ) ) {
```

This is important to ensure child theme functionality (and ability to override if needed).